README.md

```diff
@@ -21,6 +23,28 @@ p.innerHTML = "DOM in <b>Deno</b> is pretty cool";
 console.log(p.children[0].outerHTML); // "<b>Deno</b>"
 ```
 
+*minimum tsconfig.json*
+```json
+{
+  "compilerOptions": {
+    "importsNotUsedAsValues": "remove",
+    "isolatedModules": false
+  }
+}
+```
+
+To see your example in action, run:
+```sh
+deno run -c ./tsconfig.json -A --unstable ./mod.ts
+```
+
+You should see output like:
+```
+Hello from Deno!
+Deno!
+<html>DOM in <b>Deno</b> is pretty cool</html>
+```
+
```

Comment:
This isn't a bad idea, but the `tsconfig.json` isn't required here, and the command should be
```sh
deno run ./examples/basic/mod.ts
```
to align more with your example.

Reply:
Using `deno run ./examples/basic/mod.ts` would not successfully launch the example in this case :(

Reply:
I may stand corrected, hold on

Reply:
I did indeed, this should look good now